bw2io/backup.py

```diff
@@ -13,20 +13,23 @@
 
 
 def backup_data_directory(
-    timestamp: Optional[bool] = True, dir_backup: Optional[Union[str, Path]] = None
+    dir_backup: Optional[Union[str, Path]] = None,
+    timestamp: Optional[bool] = True,
+    file_suffix: Optional[str] = None,
```

Comment:
I don't think this is the right name. The initial understanding of this would be the last part of the name (e.g. `.tar.gz`, see the [pathlib docs](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix)). What about 'filename_extra' or similar instead?